src/server/models/controllers/museMetric.js

```diff
@@ -114,37 +100,14 @@ MetricManager.prototype = {
                             self.docdatabase.addItem(item, function(err){
                                 if(err){
                                     res.status(500).send({statusText: "Database error could not add new review."}); //internal error
-
-                                    var errorentry = {}; //schema for errorlog
-                                    errorentry['file'] = "museMetric.addReview";
-                                    errorentry['Status'] = 'Could not add item to metric from the museMetric.';
-
-                                    self.adderror(errorentry, function(err){ //log in error database
-                                       if (err){
-                                           callback(err);
-                                       } 
-                                    });
+                                    res.end();
+                                    return;
                                 }
-                                var accountinglog = {};
-                                accountinglog['accountID'] = AccountId;
-                                accountinglog['file'] = 'museMetric.addReview';
-                                accountinglog['status'] = 'New review added';
-
-                                self.addevent(accountinglog, function(err){ //log event data
-                                   if (err){
-                                       throw(err);
-                                   } 
-                                });
-
                                 self.accountdatabase.encodeauth(AccountId, function(err, token){
                                     if(err){
                                         throw(err);
                                     }
-                                    var headers = [];
-                                    var tokenarray = [];
-                                    tokenarray['token'] = token;
-                                    headers['auth'] = tokenarray;
-                                    responseHeader['headers'] = headers;
+                                    responseHeader['headers'] = token;

```

Comment:
Do `responseHeader['headers'] = { token }` instead. It will create an object called token in headers